include/config.h

```diff
@@ -83,7 +83,14 @@
    will be kept and written to the crash/ directory as RECORD:... files.
    Note that every crash will be written, not only unique ones! */
 
-// #define AFL_PERSISTENT_RECORD
+#define AFL_PERSISTENT_RECORD
+
+/* Adds support in compiler-rt to replay persistent records */
+#define AFL_PERSISTENT_REPLAY
```

Comment:
also this should not by default on I think. but I a second define? persistent recording only makes sense if also replay is added? so why not removing this one and just use afl_persistent_record?

Reply:
I thought persistent replay functionality might introduce some overhead in the persistent loop, and preferred to let the user choose if they want to keep a non-"replay" compiler-rt while still being able to use the record functionality in afl-fuzz.
Tbh, I did not measure if there's significant overhead added, will do and report back the  results.

If persistent replay adds significant overhead I would rather add another ifdef block to define AFL_PERSISTENT_RECORD if not defined when AFL_PERSISTENT_REPLAY is defined.
```c
#ifdef AFL_PERSISTENT_REPLAY
#ifndef AFL_PERSISTENT_RECORD
#define AFL_PERSISTENT_RECORD
#endif
#endif
```

Reply:
On a second thought is probably going to be an extremely minimal overhead, it's pretty much just one more if, I'll just use branch predictor macros, and use only one define.

Reply:
sounds good!